includes/class-sensei-utils.php

```diff
@@ -292,12 +292,38 @@ public static function sensei_text_editor( $content = '', $editor_id = 'senseite
 			'tinymce'       => array(
 				'theme_advanced_buttons1' => $buttons,
 				'theme_advanced_buttons2' => '',
+				'setup'                   => 'function (editor) {
+													// Remove placeholder on submit.
+													jQuery("#sensei-quiz-form").submit(function() {
+													    editor.dom.remove("multi-line-placeholder");
+														return true;
+													});
+													// Add placeholder on init and blur.
+													editor.on("blur init",function(){
+														if (editor.getContent() == ""){
+															editor.setContent("<p id=\'multi-line-placeholder\'>' . __( 'Your answers', 'sensei-lms' ) . '</p>");
+														}
+													});
+													// Remove placeholder on focus.
+													editor.on("focus",function(){
+														editor.dom.remove("multi-line-placeholder");
+													});
+											  }
+				',
 			),
 			'quicktags'     => false,
 		);
 
-		wp_editor( $content, $editor_id, $settings );
+		if ( false !== strpos( $input_name, 'sensei_question[' ) ) {
+
+			// Only pick the global style variables. TinyMCE loads in an iFrame, so none of our global
+			// variables are available inside it. We add them here manually.
+			$global_variables = str_replace( '"', "'", wp_get_global_stylesheet( [ 'variables' ] ) );
 
+			$settings['tinymce']['content_style'] = $global_variables . 'body.mce-content-body { background: transparent; color: var(--sensei-primary-color-global, var(--sensei-course-theme-primary-color, var(--wp--preset--color--primary, #1E1E1E))) } #multi-line-placeholder { color: #646970; }';
```

Comment:
Same question here regarding keeping the CSS in a separate file with a descriptive filename.

Reply:
Though this does make me wonder if the effort / extra code is worth such a minor change. I think simplifying by taking out the placeholder would be just fine.

Reply:
I've added the css using `file_get_contents` here https://github.com/Automattic/sensei/pull/7124/commits/3a36e00bab3ca471ffb3672c274eaf1449841ca8. I also thought about not having the placeholder, but then decided to have it in place because we don't have a label for it :p Would've been much easier